modules/manage/pages/cluster-maintenance/cluster-balancing.adoc

```diff
@@ -112,6 +116,34 @@ This requires an Enterprise license.
 This mode is not recommended for production clusters. Only set to `off` if you need to move partitions manually.
 |===
 
+== Intra-broker partition balancing
+
+In Redpanda, every partition replica is assigned to a core on a broker. While Redpanda's default <<partition-replica-balancing,partition balancing>> monitors cluster-level events such as the addition of new brokers or broker failure to balance partition assignments, it does not account for the distribution of partitions _within_ an individual broker. That means that some cores on a broker may inadvertently host many partitions of heavily used topics and cause the CPU to be xref:manage:monitoring.adoc#cpu-usage[overburdened].
+
+Additionally, when the partition rebalance moves some partitions away from a broker, the remaining partitions do not necessarily rebalance across the broker's cores. Or, if a broker's core count is increased, Redpanda does not assign any partitions to the new cores until new partitions are created or old partitions are moved out.
+
+Intra-broker partition balancing allows for dynamically reassigning partitions within a broker. If a broker's core counts increases, when the broker starts back up, Redpanda can check partition assignments across the broker's cores, and reassign partitions so that a balanced assignment is maintained across all cores. Redpanda can also check partition assignments when partitions are removed from a broker, and rebalance the remaining partitions between cores.
+
+To enable intra-broker partition balancing, configure the following broker properties:
```

Comment:
@ztlpn Are these enabled by default for new 24.2 clusters (and should we add a note regarding this)? Will these properties be made available for earlier versions as well?

Reply:
@mattschumpert is there specific guidance we want to provide regarding upgrades, or is contacting Support sufficient?